lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -500,6 +501,39 @@ def test__has_access_course_can_enroll(self):
         )
         assert not access._has_access_course(user, 'enroll', course)
 
+    @override_settings(COURSES_INVITE_ONLY=False)
+    def test__course_default_invite_only_flag_false(self):
+        """Tests that default value of COURSES_INVITE_ONLY as False."""
```

Comment:
As someone rather new to these features, the tests are somewhat unclear. Look at the above test - there's lots of comments around the assert statements & mock course creation to explain what's going on. Could you please add such comments for clarity? I wouldn't hate a more descriptive docstring, either.

Also a nit on the docstring format to do:
```
"""
Tests that default value of COURSES_INVITE_ONLY as False.
"""
```